libs/barista-components/form-field/src/form-field-control.ts

```diff
@@ -57,6 +57,9 @@ export abstract class DtFormFieldControl<T> {
   /** Whether the control is in an error state. */
   readonly errorState: boolean;
 
+  /** Whether the control is not a box control and does not use the border from the form-field. */
+  readonly nonBoxControl?: boolean;
```

Comment:
can we make this internal? 
I'd also rename this a negated boolean is always a bit problematic in terms of readability. 
I'd rather have _isBoxControl and the opposite default

Reply:
The idea with the negated name was that the value `false` or `undefined` (it's optional, to achieve backwards compatibility) lead to the same result

Reply:
I'd prefer the non negated alternative ( see another double negative :D Its already stressing me out ). 
What about making it internal?

Reply:
Cool for me :)

Whats the reasoning to make it internal?
The other props are also public. This field might also be interesting for consumer custom controls that want to support the filter field.

Reply:
I highly doubt that this is important for consumers and having it internally keeps the api surface smaller, i'd suggest make it internal now and once we get requests for accessing this feature, open it up.